terraform/modules/aws-lambda-with-cron-schedule/20-lambda.tf

```diff
@@ -0,0 +1,40 @@
+locals {
+  lambda_name_underscore = replace(lower(var.function_name), "/[^a-zA-Z0-9]+/", "_")
+}
+
+data "archive_file" "lambda" {
```

Comment:
Using libraries outside Lambda runtime won't be supported, but this is a good starting point for typical setups.